style.css

```diff
@@ -388,6 +387,73 @@ a {
     row-gap: 12px;
     padding: 24px;
   }
+
+  .popup-section {
+    position: fixed;
+    top: 0;
+    left: 0;
+    height: 100vh;
+    width: 100%;
+  }
+
+  .popup-section .background {
+    position: fixed;
+    top: 0;
+    left: 0;
+    background-color: #c1c7d0;
+    opacity: 0.6;
+    height: 100vh;
+    width: 100%;
+  }
+
+  .popup-section .tonic {
+    position: relative;
+    margin: auto;
+    transform: translateY(5%);
+    width: 90%;
+  }
+
+  .popup-section .tonic .close {
+    position: absolute;
+    right: 20px;
+    top: 20px;
+    font-size: 30px;
+    color: #7a869a;
+    border: none;
+    background: transparent;
+  }
+
```

Comment:
- [ ] The closing icon is overlapping the title of the popup window if the title is a long one. I would advise you to position the closing icon a little above the title of the popup window.

![image](https://github.com/Ridalord/my_portfolio/assets/77106962/6e4e9d5f-723d-4062-83da-9bf89373146c)